csrc/scheduler/utils.cpp

```diff
@@ -2666,6 +2666,83 @@ int64_t reorderDevicesToOuter(TensorView* tv) {
   return (int64_t)old2new.size();
 }
 
+void reorderTensorLike(
+    TensorView* target_tv,
+    const std::vector<IterDomain*>& ref) {
+  const auto& tv_loop_domain = target_tv->getLoopDomain();
+
+  IdModel id_model(target_tv->fusion(), /*build_graphs=*/false);
+  const auto& graph = id_model.buildBroadcastGraph();
```

Comment:
Out of curiosity, why are we using broadcast graph here?

Reply:
That's because I think it makes sense for ordering to consider broadcast and corresponding non-broadcast domains are mapped.